build.sbt

```diff
@@ -133,7 +133,7 @@ docker / dockerfile := {
     from("adoptopenjdk/openjdk11:alpine-slim")
     run("apk", "--no-cache", "add", "ttf-dejavu")
     add(artifact, artifactTargetPath)
-    entryPoint("java", "-Dorg.scalatra.environment=production", "-Xmx512M", "-jar", artifactTargetPath)
+    entryPoint("java", "-Dorg.scalatra.environment=production", "-Xmx1024M", "-jar", artifactTargetPath)
```

Comment:
Vet ikke om vi trenger å øke denne, men følte litt for en helgardering. Hva tenker dere?